packages/fixie/agent.yaml

```diff
@@ -0,0 +1,4 @@
+handle: ai-jsx-hello-world
+description: Hello World for AI.JSX
+more_info_url: ''
+entry_point: main:agent
```

Comment:
Do we still need this?

Reply:
I think this would probably be the way we identify the package to call to get the main component, but clearly it needs to be different than `main:agent`.